### PR TITLE
doc: add contributors section

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,3 +105,8 @@ As a minimum, any modules recommended as part of this reference architecture sho
 ## Contributing
 
 To Contribute to this project, please see the [Contributing Guide](./CONTRIBUTING.md).
+
+## Contributors
```
Comment (wtrocki) [inline README.md:109]: That might be hard to keep up to date.

Comment (mhdawson) [reply README.md:109]: @wtrocki I don't think we should remove people, as past contributions are still important.  What I'd like to have up front in the README.md  is the breadth of the contributors and different groups/scopes that they work on which have contributed to the end result.

Comment (wtrocki) [reply README.md:109]: Agreed

Comment (mhdawson) [reply README.md:109]: @wtrocki I see you approved, but thinking you should add yourself to the list as well :)

File: README.md [hunk 2]
```diff
@@ -105,3 +105,8 @@ As a minimum, any modules recommended as part of this reference architecture sho
...
+
+* Michael Dawson - Node.js lead for Red Hat and IBM
+* Lucas Holmquist - Sr. Software Engineer - Red Hat
```
Comment (jimlindeman) [inline README.md:112]: Add:
 * Jim Lindeman - Sr. Software Engineer - IBM

File: README.md (hunk 1 above)
Comment (roastlechon) [reply README.md:109]: could use this https://github.com/marketplace/actions/contribute-list

File: README.md (hunk 2 above)
Comment (domharries) [reply README.md:112]:
```suggestion
* Dominic Harries - Dev Lead at IBM Garage for Cloud
```

Comment (domharries) [reply README.md:112]: Add:

* Dominic Harries - Dev Lead at IBM Garage for Cloud

File: README.md [hunk 3]
```diff
@@ -105,3 +105,11 @@ As a minimum, any modules recommended as part of this reference architecture sho
...
 
 To Contribute to this project, please see the [Contributing Guide](./CONTRIBUTING.md).
+
+## Contributors
+
+* Michael Dawson - Node.js lead for Red Hat and IBM
+* Jim Lindeman - Sr. Software Engineer - IBM
+* Dominic Harries - Dev Lead at IBM Garage for Cloud
+* Jim Lindeman - Sr. Software Engineer - IBM
+* Jim Lindeman - Sr. Software Engineer - IBM
```
Comment (mhdawson) [inline README.md:115]:
```suggestion
* Dominic Harries - Dev Lead at IBM Garage for Cloud
```

**Comment by mhdawson** [comment]
@BethGriggs, @roastlechon, @rolfl  is if ok if I add you? If so what is the best title to use?

File: README.md [hunk 4]
```diff
@@ -105,3 +105,10 @@ As a minimum, any modules recommended as part of this reference architecture sho
 ## Contributing
 
 To Contribute to this project, please see the [Contributing Guide](./CONTRIBUTING.md).
+
+## Contributors
+
+* Dominic Harries - Dev Lead at IBM Garage for Cloud
```
Comment (BethGriggs) [inline README.md:111]:
```suggestion
* Bethany Griggs - Sr. Software Engineer - Red Hat
* Dominic Harries - Dev Lead at IBM Garage for Cloud
```

(assuming we're going alphabetical by first name :))

Comment (mhdawson) [reply README.md:111]: @BethGriggs I was planning on alphabetical

**Review by lholmquist — APPROVED** [review]
We can probably merge this now, and subsequent contributors can be added later

**Pull request merged**